src/api/app/jobs/consistency_check_job.rb

```diff
@@ -4,8 +4,11 @@
 class ConsistencyCheckJob < ApplicationJob
   queue_as :consistency_check
 
-  def perform(fix = nil)
-    init
+  def perform
+    User.get_default_admin.run_as { _perform(nil) }
```

Comment:
:thought_balloon: I don't like this. I mean I get what you are doing but can't you just wrap everything in `perform` into a `.run_as`?

Reply:
Alright we talked offline and going to refactor this later...